contracts/script/validity/OPSuccinctDGFDeployer.s.sol

```diff
@@ -11,17 +11,41 @@ import {console} from "forge-std/console.sol";
 import {ERC1967Proxy} from "@openzeppelin/contracts/proxy/ERC1967/ERC1967Proxy.sol";
 import {GameType} from "src/dispute/lib/Types.sol";
 import {IDisputeGame} from "interfaces/dispute/IDisputeGame.sol";
+import {AccessManager} from "src/lib/AccessManager.sol";
+import {LibString} from "@solady/utils/LibString.sol";
 
 contract OPSuccinctDFGDeployer is Script, Utils {
     function run() public returns (address) {
         vm.startBroadcast();
 
         OPSuccinctL2OutputOracle l2OutputOracleProxy = OPSuccinctL2OutputOracle(vm.envAddress("L2OO_ADDRESS"));
 
+        // Proposer must be permissionless or else the check in `proposeL2Output` will fail.
         l2OutputOracleProxy.addProposer(address(0));
```

Comment:
This changes the security assumptions about the `l2OutputOracleProxy`, let's discuss more.